addons/common/stringtable.xml

```diff
@@ -1151,5 +1151,13 @@
             <Chinesesimp>重量:</Chinesesimp>
             <Chinese>重量:</Chinese>
         </Key>
+        <Key ID="STR_ACE_Common_AllowFadeMusic">
+            <English>Allow muting music</English>
```

Comment:
Are we muting the music though? I think it'd be a bit clearer like "Allow music volume adjustment"

Reply:
You're right. I read `0 fadeMusic` wrong.